src/app/services/org-account-helper/org-account-helper.coffee

```diff
@@ -13,7 +13,7 @@ angular.module 'mnoEnterpriseAngular'
       # Is an up to date account required to allow app management and is the account past due?
       paymentRequired = MnoeConfig.isCurrentAccountRequired() && selectedOrg.organization.in_arrears
       # Are billing details required and are they present?
-      detailsRequired = MnoeConfig.areBillingDetailsRequired() && _.isEmpty(selectedOrg.credit_card)
+      detailsRequired = MnoeConfig.areBillingDetailsRequired() && !selectedOrg.credit_card.presence
```

Comment:
Does that work in JS?
Or is this a field exposed by the backend?

Reply:
It's a field exposed by the backend. I saw this was done in the admin panel and ran into an issue where the credit card isn't returned in the response for a member or manager user. So I copied the pattern here. For member or manager users the credit card response only contains this attribute.

Reply:
Updated to only show id attr of credit cards when user is a member or manager.